### DISCUSSION TITLE
Display P95 and mean time in Grafana from micrometer

### DISCUSSION THREAD

**Discussion by Estenden** [quarkusio/quarkus#30996] [Q&A] [answered]
Hello,

I am facing issue regarding Grafana, Prometheus and Micrometer.
The P95 and mean times are not exported to Prometheus so It is not displayed in Grafana.
Only the count, sum and max values are present in the micrometer export.
Can you tell me how can I configure Micrometer to export those values into Prometheus and then display It in Grafana?

[a comment by an automated account is not shown]

**Comment by ebullient** [comment, marked as answer]
Micrometer uses MeterFilters to selectively enable gathering of additional metrics. There is a storage cost, so these additional metrics are not enabled by default.

The syntax is here: 
https://micrometer.io/docs/concepts#_histograms_and_percentiles

There is an example of CDI-based registration/injection of MeterFilters here: 
https://quarkus.io/guides/micrometer#using-meterfilter-to-configure-metrics

**Reply by Estenden** [reply]
Thank you. I managed to get the data in Grafana.
